docs/static/security/es-security.asciidoc

```diff
@@ -0,0 +1,76 @@
+[discrete]
+[[es-security-on]]
+== {es} security on by default
+
+{es} {ref}/configuring-stack-security.html[security is on by default] starting in 8.0. 
+{es} is the certificate authority (CA), and it generates its own self-signed Secure Sockets Layer (SSL) certificate at startup. 
+{ls} must establish a secure connection using the SSL certificate before it can transfer data. 
+Our hosted {ess} can handle these details for you. 
+
+
+.Hosted {ess} simplifies security
+[NOTE]
+=====
+Our hosted {ess} on Elastic Cloud simplifies safe, secure communication between {ls} and {es}. 
+When you configure the elasticsearch output plugin to use <<plugins-outputs-elasticsearch-cloud_id,`cloud_id`>> with either the <<plugins-outputs-elasticsearch-cloud_auth,`cloud_auth` option>> or the <<plugins-outputs-elasticsearch-api_key,`api_key` option>>, no additional SSL configuration steps are needed.
+
+Examples:
+
+* `output {elasticsearch { cloud_id => "" cloud_auth => "" } }`
+* `output {elasticsearch { cloud_id => "" api_key => "" } }``
+
+For more details, check out the
+{logstash-ref}/connecting-to-cloud.html[Logstash-to-Cloud documentation].
+
+{ess-leadin-short}
+=====
+
+[discrete]
+[[es-security-onprem]]
+=== Secure communication with an on-premise {es} cluster
+
+If you are running {es} on your own hardware, you need to complete a few more steps to establish secure communication between {ls} and {es}.
+You need to: 
+
+* Copy the self-signed certificate from {es} and save it to {ls}
+* Configure the elasticsearch-output plugin to use the certificate
+
+[discrete]
+[[es-sec-copy-cert]]
+==== Copy and save the certificate
+
+{es} generates a self-signed certificate when it starts.
+{ls} needs its own copy of the self-signed certificate from the {es} cluster so that it can validate the certificate presented by {es}.
+
+Copy the self-signed certificate from {es} to {ls}. 
```

Comment:
ToDo: Add more details (and/or links) on how/where the user finds/obtains the ES cert.